apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/BeneficiaryTransformer.java

```diff
@@ -55,21 +55,29 @@ private static Patient transform(
     patient.addIdentifier(
         TransformerUtils.createIdentifier(
             CcwCodebookVariable.BENE_ID, beneficiary.getBeneficiaryId()));
+    patient
+        .addIdentifier()
+        .setSystem(TransformerConstants.CODING_BBAPI_BENE_HICN_HASH)
+        .setValue(beneficiary.getHicn());
+
```

Comment:
Why add HICN hash here? Or why not add MBI hash as well?

Reply:
I added this back related to the Revert "BLUEBUTTON-1679 Remove hicn-hash identifier from Patient Resource" #174 which hasn't been merged in to master yet. That was related to a change in priority for when that task is due.  I'll create another PR related to that ticket later.

The MBI hash identifier is being added as part of another PR:  BLUEBUTTON-1516: Search patients by MBI hash https://github.com/CMSgov/beneficiary-fhir-data/pull/172/commits/300ac036ffc1cd4fd95815d770b20f71880972fa#